contracts/utils/math/Math.sol

```diff
@@ -13,6 +13,72 @@ library Math {
         Zero // Toward zero
     }
 
+    /**
+     * @dev Returns the addition of two unsigned integers, with an overflow flag.
+     *
+     * _Available since v3.4._
```

Comment:
These "available since" comments should probably be updated to say 5.0.

Reply:
resoled